chia/policy/fee_estimator_demo.py

```diff
@@ -0,0 +1,35 @@
+from chia.policy.fee_estimation import FeeBlockInfo, FeeMempoolInfo
+from chia.policy.fee_estimator import FeeEstimatorConfig
+from chia.types.mempool_item import MempoolItem
+from chia.util.ints import uint64
+
+MIN_MOJO_PER_COST = 5
+
+
+def demo_fee_rate_function(cost: int, time_in_seconds: int) -> uint64:
+    return uint64(cost * MIN_MOJO_PER_COST * max((3600 - time_in_seconds), 1))
+
+
+class FeeEstimatorDemo:  # FeeEstimatorInterface Protocol
```

Comment:
Should this be adopting the FeeEstimatorInterface in some manner? I'm not familiar enough with python's best practices in this area.

Reply:
Ah, the way this works is:

If you declare a variable elsewhere as type FeeEstimatorInterface, python will throw if the object you pass does not conform to the protocol object. Unfortunately, that is a bit implicit.

However, I think that in addition to this python Protocol, we should also have Estimators that plug into our infrastructure inherit from an object that will enforce some of the algorithmic things we expect.

Normally I would avoid inheritance for behaviour sharing, but I think it might help to enforce some of our invariants (e.g. no negative estimates).